pkg/quay/quay.go

```diff
@@ -57,10 +69,11 @@ func (q *Quay) ListRepositories() ([]string, error) {
 		return reposToSync, microerror.Mask(err)
 	}
 
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", q.token))
+
 	query := req.URL.Query()
 	query.Add("last_modified", "true")
 	query.Add("starred", "false")
-	query.Add("public", fmt.Sprintf("%t", publicImagesOnly))
```

Comment:
Isn't it better to add the if here?

```
if !q.includePrivateRepositories {
	query.Add("public", fmt.Sprintf("%t", publicImagesOnly))
}
```

Reply:
That query param has no effect unfortunately

Reply:
:(